x/validators/model.go

```diff
@@ -69,13 +70,13 @@ func GetAccounts(bucket orm.Bucket, kv weave.KVStore) (*Accounts, error) {
 	}
 
 	if res == nil {
-		return nil, ErrNotFound("account")
+		return nil, errors.NotFoundErr.New("account not found")
```

Comment:
I think this is good, but just "account". I think the not found error adds the other part.
We want `account: not found` not `account not found: not found`